assets/js/components/Work/UIWorkCardItem.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import { setVisibilityClass } from "../../services/helpers";
+
+const WorkCardItem = ({ work }) => {
+  const fileSetsToDisplay = 5;
+
+  return (
+    <div className="card is-shadowless" data-testid="ui-workcard">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <Link to={`/work/${work.id}`}>
+            <img
+              src={`${work.representativeImage}/full/1280,960/0/default.jpg`}
+              data-testid="image-work"
+              alt={work.title}
+              onError={(e) => {
+                e.target.src = "/images/1280x960.png";
+              }}
+            />
+          </Link>
+        </figure>
+      </div>
+      <div className="card-content">
+        <h3 className="title is-size-4">
+          {work.accessionNumber} <span className="tag">{work.workType}</span>
```

Comment:
Let's display a Work title (located in DescriptiveMetadata), by default if one exists, and "Untitled" as a fallback.  Then let's move Accession Number into the `<dd>` list I mentioned in the screen shot notes.